db_definitions.sql

```diff
@@ -50,3 +50,11 @@ ALTER TABLE `asset_folder_mapping` ADD FOREIGN KEY (`asset_id`) REFERENCES `asse
 
 ALTER TABLE `asset_folder_mapping` ADD FOREIGN KEY (`folder_id`) REFERENCES `folder` (`id`);
 
+/*Add some dummy data*/
+INSERT INTO `user`(email,pw) VALUES ("test_user@test.test","hunter2");
+
+INSERT INTO `asset`(name,physical_file_name,uploader,physical_file_size,physical_file_type) VALUES ("testAsset","some_image.jpg",1,3,"jpg");
```

Comment:
Didn't occur to me until just now, but we should make it so `physical_file_size` can accept a decimal.

Reply:
Or I guess we could say that number is bytes. I was thinking it would represent like megabytes, but that's dumb because not everything is at least a megabyte. So nevermind...